website/views/layouts/layout.ejs

```diff
@@ -55,7 +55,8 @@
   </head>
   <body>
     <div purpose="page-wrap">
-      <div style="max-width: 1248px; height: 94px;" class="<%= isHomepage ? 'homepage-header' : 'header' %> container-fluid d-flex justify-content-between align-items-center pt-3 pb-3 px-3 px-md-4" purpose="page-header">
+      <div class="<%= isHomepage ? 'homepage-header' : 'header' %>" purpose="page-header">
+        <div style="max-width: 1248px; height: 80px;" class="container-fluid d-flex justify-content-between align-items-center pt-3 pb-3 px-3 px-md-4">
```

Comment:
Would you update the indentation of the stuff inside as well, since there's an extra div now?  It'll mean more redlines in this diff, but will be easier to maintain and see what's going on going forward

Also please double-check to make sure closing `</div>` is in the correct place, and ends up with the right indentation

Reply:
Updated. Not sure what GitHub etiquette is here. Should I leave resolving the conversation to your (the person who opened it)?